README.md

```diff
@@ -3,36 +3,130 @@
+converging multivendor hardware equipment. By exposing a standards-based programming interface, it enables easy and
+secure management of wide range of multivendor IT infrastructure distributed across multiple data centers.
+
+ODIMRA framework comprises the following two components.
+
+- The resource aggregation function (the resource aggregator)
+
+  The resource aggregation function is the single point of contact between the northbound clients and the
+  southbound infrastructure. Its primary function is to build and maintain a central resource inventory. It exposes
+  Redfish-compliant APIs to allow northbound infrastructure management systems to:
+    - Get a unified view of the southbound compute, local storage, and Ethernet switch fabrics available in the
+      resource inventory.
+    - Gather crucial configuration information about southbound resources.
+    - Manipulate groups of resources in a single action.
+    - Listen to similar events from multiple southbound resources.
+	
+- One or more plugins
+
+  The plugins abstract, translate, and expose southbound resource information to the resource aggregator through
+  RESTful APIs. HPE Resource Aggregator for ODIM supports:
+ 
+    - Generic Redfish plugin for ODIM (GRF): Generic Redfish plugin that can be used as a plugin for any Redfishcompliant
+      device.
+	- Plugin for unmanaged racks (URP): Plugin that acts as a resource manager for unmanaged racks. 
+    - Integration of additional third-party plugins.  
+
+
+
+
+##  Resource Aggregator for ODIM logical architecture
+
+Resource Aggregator for ODIM framework adopts a layered architecture and has many functional layers.
+
+
+The following figure shows these functional layers of Resource Aggregator for ODIM deployed in a data center.
+
+![ODIM_architecture](images/arch.png)
+
+- **API layer**
+
+
+This layer hosts a REST server which is open-source and secure. It learns about the southbound resources from
+the plugin layer and exposes the corresponding Redfish data model payloads to the northbound clients. The
+northbound clients communicate with this layer through a REST-based protocol that is fully compliant with
+DMTF's Redfish® specifications (Schema 2020.3 and Specification 1.11.1).
+The API layer sends user requests to the plugins through the aggregation, the event, and the fabric services.
+
+- **Services layer**
+
+
+The services layer is where all the services are hosted. This layer implements service logic for all use cases
+through an extensible domain model (Redfish Data Model). Requests coming from the API layer and the
+responses coming from the plugin layer are mapped to the actual end resources in this layer. It maintains the state
+for event subscriptions, credentials, and tasks. It also hosts a message bus called the Plug-in Message Bus (PMB).
+
+![Redfish_data_model](images/redfish_data_model.png)
```

Comment:
This link is now working.

Reply:
It will work once it is merged to this branch.